ui/js/controllers/logviewer.js

```diff
@@ -283,6 +293,18 @@ logViewerApp.controller('LogviewerCtrl', [
             });
         }
 
+        function drawSelectedLines(data) {
+            if (data.length === 0) {
+                return;
+            }
+            //TODO: Test $scope.selectedBegin and $scope.selectedEnd
+            var i = $scope.selectedBegin;
+            var end = $scope.selectedEnd;
+            for (i ; i <= end ; i++) {
```

Comment:
There shouldn't be a space between the `i` and `end` vars, and their semicolons.